inst/resources/html/paged.html

```diff
@@ -7,7 +7,7 @@
 <meta charset="utf-8">
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <meta name="generator" content="pandoc" />
-<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta name="viewport" content="width=device-width">
```

Comment:
BTW, I have used `initial-scale=1` for a long time: https://github.com/rstudio/rmarkdown/commit/98fd46a5b02228bbebf8111fbe09c3fa6a8158e4 and haven't run into the zooming problems, although I don't fully understand it.

Reply:
AFAIK, this is because `rmarkdown::html_document` use a responsive design. Here, with paged.js, the design is non responsive (but we could make it responsive with JS if you want).
In [this article](http://webdesignerwall.com/tutorials/viewport-meta-tag-for-non-responsive-design), it is explained that 

> A common mistake is that people often apply initial-scale=1 on non-responsive design. This will make the page render at 100% without scaling. If your design is not responsive, users would have to pan around or zoom out to see the full page.

Reply:
Thanks for the explanation!